_posts/2023-11-02-election-competitiveness-in-charlotte-north-carolina/election-competitiveness-in-charlotte-north-carolina.Rmd

```diff
@@ -0,0 +1,76 @@
+---
+title: "Election Competitiveness in Charlotte, North Carolina"
+description: |
+  Analyzing changes in competitiveness for mayoral and city council elections from 2015 to 2022 in Charlotte, North Carolina.
+author: Angeline Zhao
+date: 2023-11-02
+output:
+  distill::distill_article:
+    self_contained: false
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+## City Council in Charlotte
+
+The Charlotte City Council is made of the Mayor, four at-large city council members, and one council member for each of the seven city council districts. Currently, two members of the city council are Republicans and the remaining are Democrats.
+
+For the demographic breakdown, Charlotte is roughly 40% White, 33% Black, 16% Hispanic, and 7% Asian. For the partisan breakdown, Charlotte voted 68% Democrat and 32% Republican in the the latest 2021 Mayoral Election.
```

Comment:
Make sure to cite where you got these demographic and electoral numbers from.

Reply:
Nitpicky but maybe reword as "The demographic breakdown in Charlotte is approximately 40% White, ..."